+bids/File.m

```diff
@@ -707,6 +739,48 @@ function check_required_entities(obj)
 
     end
 
+    % Functions related to metadata manipulation
+
+    function obj = metadata_update(obj, varargin)
+      % Update stored metadata with new values passed in varargin,
+      % which can be either a structure, or pairs of key-values.
+      %
+      % See also
+      %    bids.util.update_struct
+      %
+      % USAGE::
+      %
+      %  f = f.metadata_update(key1, value1, key2, value2);
+      %  f = f.metadata_update(struct(key1, value1, ...
+      %                        key2, value2));
+
+      obj.metadata = bids.util.update_struct(obj.metadata, varargin{:});
+    end
+
+    function out_file = metadata_export(obj, varargin)
```

Comment:
```suggestion
    function out_file = metadata_write(obj, varargin)
```

Maybe write to stay consistent with tsvwrite? Or `encode` to stay consistent with `jsonencode`? WDYT?